gflib/text.h

```diff
@@ -269,20 +269,13 @@
 
 #define TEXT_SPEED_FF 0xFF
 
-enum
-{
-    COLOR_FOREGROUND,
-    COLOR_SHADOW,
-    COLOR_BACKGROUND
-};
-
 enum
 {
     FONTATTR_MAX_LETTER_WIDTH,
     FONTATTR_MAX_LETTER_HEIGHT,
     FONTATTR_LETTER_SPACING,
     FONTATTR_LINE_SPACING,
-    FONTATTR_STYLE,
+    FONTATTR_UNKNOWN,   // dunno what this is yet
```

Comment:
Why undo this change? Is it still unknown?

Reply:
Because it's a 10k PR and we need to limit scope if it's going to ever happen.

Reply:
It was added in #1149, I don't see any reason to undo that...? (unless it's completely wrong)

Reply:
This is a draft PR started by reverting the listed PRs in full, please refrain from commenting on specific reversions until it's complete

Reply:
I'm mostly asking why are these seemingly okay changes being reverted, seems pretty counterintuitive to replace named fields with "UNKNOWN"

Reply:
As I said, the PR will be pared down to useful changes. The PR is a draft, it's incomplete, and the field in question is never read regardless, so naming it doesn't matter much whether it's kept or not

Reply:
Ohhh, gotcha.